### PR TITLE
keyboard-a11y: Add UI for capslock-beep

### PR DESCRIPTION
**Pull request by cwendling**
Depends on https://github.com/mate-desktop/mate-desktop/pull/450.  See also https://github.com/mate-desktop/mate-settings-daemon/pull/330.

### CODE REVIEW

**Comment by cwendling** [comment]
Travis fails because of the mate-desktop dependency that isn't satisfied there.

File: capplets/keyboard/ui-a11y.suppr [hunk 1]
```diff
@@ -1,3 +1,5 @@
+mate-keyboard-properties-a11y-notifications.ui://GtkLabel[@id='label1'] orphan-label
+mate-keyboard-properties-a11y-notifications.ui://GtkLabel[@id='label2'] orphan-label
```
Comment (cwendling) [inline capplets/keyboard/ui-a11y.suppr:2]: This is not nice indeed, but I plan on rewriting that UI file later on not to have all these issues.

File: .build.yml [hunk 2]
```diff
@@ -171,11 +171,12 @@ before_scripts:
...
-  - if [ ! -f mate-desktop-1.23.2.tar.xz ];then
-  -     curl -Ls -o mate-desktop-1.23.2.tar.xz https://github.com/mate-desktop/mate-desktop/releases/download/v1.23.2/mate-desktop-1.23.2.tar.xz
+  - MATE_DESKTOP_VERSION=1.25.0
+  - if [ ! -f mate-desktop-${MATE_DESKTOP_VERSION}.tar.xz ];then
+  -     curl -Ls -o mate-desktop-${MATE_DESKTOP_VERSION}.tar.xz https://github.com/mate-desktop/mate-desktop/releases/download/v${MATE_DESKTOP_VERSION}/mate-desktop-${MATE_DESKTOP_VERSION}.tar.xz
   - fi
-  - tar xf mate-desktop-1.23.2.tar.xz
-  - cd mate-desktop-1.23.2
+  - tar xf mate-desktop-${MATE_DESKTOP_VERSION}.tar.xz
+  - cd mate-desktop-${MATE_DESKTOP_VERSION}
```
Comment (sc0w) [inline .build.yml:179]: I think nowadays all this code it isn't needed

@raveit65 ping

Comment (cwendling) [reply .build.yml:179]: Well, without updating it the automated builds failed because they were using a too old version of mate-desktop.  Now they pass.

Comment (sc0w) [reply .build.yml:179]: ah ok, understood, sorry for the noise

**Comment by sc0w** [comment]
you can also change these lines with new `mate-desktop` version in `build.yml`

`  -     cppcheck --enable=warning,style,performance,portability,information,missingInclude -i'mate-desktop-1.23.2' -i'mate-settings-daemon-1.23.1' `.

`  -   cppcheck --xml --output-file=cppcheck.xml --enable=warning,style,performance,portability,information,missingInclude -i'mate-desktop-1.23.2' -i'mate-settings-daemon-1.23.1' .`

**Comment by cwendling** [comment]
> you can also change these lines with new `mate-desktop` version in `build.yml` […]

Good catch.  I just did, but I also had to rebase on master as there have been conflicting changes in this portion since I started this branch.

**Comment by cwendling** [comment]
Merging as there have been 2 approvals for a week.

**Comment by raveit65** [comment]
@cwendling 
> Merging as there have been 2 approvals for a week.

This one is waiting for reviewers since 5 weeks.
https://github.com/mate-desktop/mate-applets/pull/483
Before we break accessibility support....

**Comment by cwendling** [comment]
@raveit65 I plan on testing it (looking at the changes look good, but I didn't test yet), but I won't have the proper setup until probably 2 weeks, as mentioned on another of my PRs, so I'll do that then.

About this one, as I didn't see mentions why not to merge it and there were approvals I went ahead, but if I shouldn't have I can roll it back, and/or I'll refrain from merging so next time.  Do we have rules for that BTW?  I often saw this kind of process lately, e.g. me reviewing and approving a PR and the author merging right afterward (which seemed sound to me), so I though it was alright, but I don't want to overstep anything.

**Comment by raveit65** [comment]
I didn't say anything :)
Normally 2 approvals are OK, and the one who merge is responsible that the change is fine.
I didn't review this one for myself because there are about 50 open PRs in my mailbox and that's not all.
Those are really enough to review for whole 2020 :)
Usually i am working on the oldest PRs to avoid that we forget one.
So, it is possible that it takes 1 tor 3 months before i reach newer PRs.

**Pull request merged**